lib/ex_webrtc/peer_connection.ex

```diff
@@ -89,147 +118,292 @@ defmodule ExWebRTC.PeerConnection do
-          :ok | {:error, atom()}
+          :ok | {:error, term()}
   def set_remote_description(peer_connection, description) do
     GenServer.call(peer_connection, {:set_remote_description, description})
   end
 
-  @spec get_remote_description(peer_connection()) :: SessionDescription.t() | nil
-  def get_remote_description(peer_connection) do
-    GenServer.call(peer_connection, :get_remote_description)
+  @doc """
+  Creates an SDP offer.
+
+  For more information, refer to the [RTCPeerConnection: createOffer() method](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createOffer).
+  """
+  @spec create_offer(peer_connection(), restart_ice?: boolean()) ::
+          {:ok, SessionDescription.t()} | {:error, term()}
+  def create_offer(peer_connection, options \\ []) do
+    GenServer.call(peer_connection, {:create_offer, options})
   end
 
-  @spec get_current_remote_description(peer_connection()) :: SessionDescription.t() | nil
-  def get_current_remote_description(peer_connection) do
-    GenServer.call(peer_connection, :get_current_remote_description)
+  @doc """
+  Creates an SDP answer.
+
+  For more information, refer to the [RTCPeerConnection: createAnswer() method](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createAnswer).
+  """
+  @spec create_answer(peer_connection()) :: {:ok, SessionDescription.t()} | {:error, term()}
+  def create_answer(peer_connection) do
+    GenServer.call(peer_connection, :create_answer)
   end
 
-  @spec add_ice_candidate(peer_connection(), ICECandidate.t()) ::
-          :ok | {:error, :no_remote_description}
+  @doc """
+  Adds a new remote ICE candidate.
+
+  For more information, refer to the [RTCPeerConnection: addIceCandidate() method](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/addIceCandidate).
+  """
+  @spec add_ice_candidate(peer_connection(), ICECandidate.t()) :: :ok | {:error, term()}
   def add_ice_candidate(peer_connection, candidate) do
     GenServer.call(peer_connection, {:add_ice_candidate, candidate})
   end
 
-  @spec get_transceivers(peer_connection()) :: [RTPTransceiver.t()]
-  def get_transceivers(peer_connection) do
-    GenServer.call(peer_connection, :get_transceivers)
-  end
+  @doc """
+  Adds a new transceiver.
 
+  For more information, refer to the [RTCPeerConnection: addTransceiver() method](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/addTransceiver).
+  """
   @spec add_transceiver(
           peer_connection(),
           RTPTransceiver.kind() | MediaStreamTrack.t(),
-          transceiver_options()
+          direction: RTPTransceiver.direction()
         ) :: {:ok, RTPTransceiver.t()}
   def add_transceiver(peer_connection, kind_or_track, options \\ []) do
     GenServer.call(peer_connection, {:add_transceiver, kind_or_track, options})
   end
 
+  @doc """
+  Sets the direction of transceiver specified by the `id`.
```

Comment:
Take a look at other functions too

```suggestion
  Sets the direction of transceiver specified by the `transceiver_id`.
```